tartufo/scanner.py

```diff
@@ -136,13 +136,13 @@ def included_paths(self) -> List[Pattern]:
         :rtype: List[Pattern]
         """
         if self._included_paths is None:
+            patterns = self.global_options.include_path_patterns or ()
             if self.global_options.include_paths:
-                self._included_paths = config.compile_path_rules(
-                    self.global_options.include_paths.readlines()
-                )
+                patterns += tuple(self.global_options.include_paths.readlines())
```

Comment:
These should probably be lists, since that's the type that this function returns. Either that or maybe it makes sense for this to change to a `Tuple` return value?